src/test/java/teammates/test/cases/pagedata/StudentFeedbackResultsPageDataTest.java

```diff
@@ -96,19 +96,37 @@ public void testAll() throws EntityDoesNotExistException {
         assertEquals("", questionBundle1.getQuestionDetails().getAdditionalInfo());
         assertEquals("", questionBundle2.getQuestionDetails().getAdditionalInfo());
 
-        assertNotNull(questionBundle1.getResponseTables());
-        assertNotNull(questionBundle2.getResponseTables());
+        ______TS("giver & receiver in self/others response tables");
 
-        assertEquals("You", questionBundle1.getResponseTables().get(0).getRecipientName());
+        assertNotNull(questionBundle1.getSelfResponseTables());
+        assertNotNull(questionBundle1.getOthersResponseTables());
+        assertNotNull(questionBundle2.getSelfResponseTables());
+        assertNotNull(questionBundle2.getOthersResponseTables());
 
-        assertNotNull(questionBundle1.getResponseTables().get(0).getResponses());
+        assertTrue(questionBundle1.getSelfResponseTables().get(0).isGiverNameYou());
+        assertEquals("You", questionBundle1.getSelfResponseTables().get(0).getRecipientName());
 
-        assertEquals("You", questionBundle1.getResponseTables().get(0).getResponses()
+        assertEquals(0, questionBundle1.getOthersResponseTables().size());
+        assertNotNull(questionBundle1.getSelfResponseTables().get(0).getResponses());
```

Comment:
- Any reason of putting these assertion together? One is testing getOthersResponseTables while the other is testing `getSelfResponseTables`.

Reply:
This is just because I was still following the original test cases, where the cases fall into two tables. Will reorder it.